audit/projects/kubernetes-public/secrets/spiffxp-test/description.json

```diff
@@ -1,7 +0,0 @@
-{
-  "createTime": "2021-04-08T19:41:00.816372Z",
-  "name": "projects/127754664067/secrets/spiffxp-test",
-  "replication": {
-    "automatic": {}
-  }
-}
```

Comment:
Expected, this was me removing a dummy secret I had manually created for testing